examples/text_labels_and_annotations/usetex_baseline.py

```diff
@@ -1,8 +1,10 @@
 """
 ====================
-Usetex Baseline Test
+Usetex Baseline Demo
```

Comment:
This looks more like somebody has created a test image so that they can visually check that the baseline is correct. I don't think this has any instructional value. Should we remove this and instead make a real test out of it (if we don't have one)?